video/quickstart.js

```diff
@@ -17,15 +17,15 @@
 
 // [START videointelligence_quickstart]
 // Imports the Google Cloud Video Intelligence library
-const Video = require('@google-cloud/videointelligence').v1beta1();
+const Video = require('@google-cloud/video-intelligence');
 
 // Instantiates a client
-const video = Video.videoIntelligenceServiceClient({
+const video = Video({
   projectId: process.env.GCLOUD_PROJECT // Replace with your Google Cloud project ID
 });
 
 // The GCS filepath of the video to analyze
-const gcsUri = 'gs://demomaker/volleyball_court.mp4';
+const gcsUri = 'gs://demomaker/tomatoes.mp4';
```

Comment:
No action required, but I'm using the ML videos from the video landing page, examples here:

https://github.com/GoogleCloudPlatform/python-docs-samples/tree/video_v1beta1/video/cloud-client/analyze